mephisto/server/blueprints/abstract/static_task/static_blueprint.py

```diff
@@ -135,6 +162,13 @@ def add_args_to_group(cls, group: "ArgumentGroup") -> None:
             default=1,
             type=int,
         )
+        group.add_argument(
+            "--subtasks-per-unit",
+            dest="subtasks_per_unit",
+            type=int,
+            default=5,
+            help="number of subtasks/comparisons to do per unit",
+        )
```

Comment:
This last part is probably meant for your multi-subtask-per-unit blueprint.

Reply:
Yeah let me know: I would say that having subtasks is a pretty common ask for us, so that's why I didn't put it in the task-specific blueprint I created.